### PR TITLE
[stable/rocketchat] Fixed TLS Ingress, updated  app and deps to latest version

### PR DESCRIPTION
**Pull request by verwilst**
Signed-off-by: Bart Verwilst <bart@verwilst.be>

**What this PR does / why we need it**: TLS didn't work. Updated mongodb to 4.3.x and app to latest stable.

### CODE REVIEW

File: stable/rocketchat/requirements.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 dependencies:
 - name: mongodb
-  version: 4.2.2
+  version: 4.3.x
```
Comment (davidkarlsen) [inline stable/rocketchat/requirements.yaml:3]: See syntax for versionranges: https://github.com/helm/helm/blob/master/docs/chart_best_practices/requirements.md

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/rocketchat/requirements.yaml (hunk 1 above)
Comment (verwilst) [reply stable/rocketchat/requirements.yaml:3]: Sure, and looking further down that page: https://github.com/Masterminds/semver#checking-version-constraints :-) .x is a valid wildcard ;-)

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**